subsys/bluetooth/shell/bt.c

```diff
@@ -1313,10 +1390,58 @@ static int cmd_auth_passkey(const struct shell *shell,
 }
 #endif /* CONFIG_BT_SMP) || CONFIG_BT_BREDR */
 
+/* Dummy command to ensure that help is printed. */
```

Comment:
This is not needed. Help is printed when command does not have a handler.

Reply:
removed